content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,569 @@
+in effect.
+
+To execute the experiments in parallel, please see the next section.
+
+### Running Experiments in Parallel
+
+DVC allows to run the experiments in parallel by specifying the number of
+experiment processes:
+
+```dvc
+$ dvc exp run --run-all --jobs 4
+```
+
+Each experiment is run _serially_ in its own temporary directory. If there are
+common stages across these experiments that need to be run, each experiment runs
+those separately. For example, for a pipeline composed of the stages
+`A -> B -> C`, if `dvc exp run --queue -S param=value1` invalidates stage `A`,
+all the pipeline is run in all experiments.
+
+⚠️ Parallel runs are experimental and may be unstable at this time. ⚠️ Make sure
+you're using a number of jobs that your environment can handle (no more than the
+CPU cores).
+
+### Experiments outside of the Workspace
+
+If you want to isolate the experiments in their own directory, you can do so by
+`--temp` flag. This allows to continue your work while running the experiment.
+
+```dvc
+$ nohup dvc exp run --temp &
+[1] 30473
+nohup: ignoring input and appending output to 'nohup.out'
+```
+
+The command checks out all DVC-tracked files and Git-tracked files into a
+temporary directory under `.dvc/tmp/exps/` and runs the experiment there. It
+creates a `nohup.log` file in the project directory. If you want to specify the
+output filename, you can use redirection.
+
+```dvc
+$ nohup dvc exp run --temp > my-experiment-$(date +"%F-%H-%M-%S").log
+```
+
+> ⚠️ Note that only tracked files and directories will be included in
+> `--queue/temp` experiments. To include untracked files, stage them with
+> `git add` first (before `dvc exp run`). Feel free to `git reset` them
+> afterwards. Git-ignored files/dirs are explicitly excluded from runs outside
+> the workspace to avoid committing unwanted files into experiments.
+
+### Cleaning Up the Experiment Queue
+
+You can use `dvc exp remove --queue` or `dvc exp gc --queued` to remove the
+experiments from the queue. For detailed information see the [section on
+Cleaning-Up the Experiments][ug-clean-up].
+
+[ug-clean-up]: /doc/user-guide/experiment-management/cleaning-up-experiments
+
+### How are experiments queued?
+
+The experiments are created similar to
+[Git stash](https://www.git-scm.com/docs/git-stash) when queued. The last
+experiment is found in `.git/refs/exps`, and the earlier ones are in the reflog.
+During `--run-all`, these references are checked out to `.dvc/exps/temp/` and
+run there.
+
+## Checkpoints
+
+To track successive steps in a longer or deeper <abbr>experiment</abbr>, you can
+register checkpoints from your code. Each `dvc exp run` will resume from the
+last checkpoint.
+
+Checkpoints provide a way to train models iteratively, keeping the metrics
+associated with each epoch.
+
+### Creating Checkpoints
+
+#### Adding Checkpoints to the Pipeline
+
+There are various ways to add checkpoints to a project. In common, these all
```

Comment:
We have a separate Checkpoints guide. Let's only link there from here?

The work you've done here could be merged into that guide if it's a significant improvement though, but as a separate PR I think (BTW I'm reviewing that guide too).

Reply:
That guide is more like a tutorial and I think better be replaced by this section. It also has to tell the experiments because these two topics cannot be clearly separated. When we have a GS page for the checkpoints, we can remove that page I think.

Reply:
Yes it's like a tutorial so it needs revising, not discarding it and replacing it by a relatively short section here. Checkpoints definitely deservers a guide don't you think? But that's a separate problem, not necessarily in the scope of this "running experiments" guide.

Reply:
Checkpoints are extensions to experiments, they cannot be discussed without first adding a stage, experimenting it, etc. That tutorial mostly deals with the setup of checkpoints. It doesn't even tell different ways to use it. I believe the content here covers more about the checkpoints itself than the tutorial.

Reply:
> That tutorial mostly deals with the setup of checkpoints. It doesn't even tell different ways to use it

Again, it's separate problem to improve that other guide. If we have a checkpoints guide, let's put everything-checkpoints there and just mention & link from here 🙂

Reply:
p.s. working on that ⏳

Reply:
Done in d6b4d5cf. Just left the section on **running checkpoints** that links to the full guide.